decred/decred/dcr/account.py

```diff
@@ -868,6 +870,7 @@ def blob(acct):
         """Satisfies the encode.Blobber API"""
         return (
             BuildyBytes(0)
+            .addData(encode.intToBytes(acct.idx))
```

Comment:
Modified the encoding without updating the version. Sync from mnemonic seed. I am planning on stopping doing this soon, maybe after we resolve #102.